digital_twin_advanced.py

```
@@ -0,0 +1,453 @@
+        config["ollama_model"] = model if model else "llama3.2"
+    
+    with open("twin_config.json", 'w', encoding='utf-8') as f:
+        json.dump(config, f, indent=2)
+    
+    print("\n✅ LLM configuratie opgeslagen!")
+    return config
+
+
+def main():
+    """Hoofdprogramma"""
+    import sys
```

Ruff (0.14.10)

Redefinition of unused `sys` from line 11

Remove definition: `sys`

(F811)